src/app/editor/components/node-editor/node-editor.component.ts

```diff
@@ -143,8 +146,30 @@ export class NodeEditorComponent implements OnInit, OnDestroy {
         return !!this.node && !/\.0$/.test(this.node.version);
     }
 
+    // carries on the saving process.
+    // displays a loading overlay if binary fields has to be uploaded
```

Comment:
COuld you use the /* */ comment style for methods? And as much as possible proper English grammar / punctuation.